### PR TITLE
Delete old actions

### PR DESCRIPTION
**Pull request by anton-br**
Some useless functions/classes have been deleted. Find full list below:
* class IndexTracker
* def show_research
* def draw_histogram
* def items_viewer
* def to_2d
* def check_unique_fieldrecord_across_surveys
* def print_results
* def calc_v_rms

If i've deleted something important (or warm to your heart), let me know.

### CODE REVIEW

**Review by a-arefina — COMMENTED** [review]
Please keep `show_research` or move it to batchflow

**Comment by anton-br** [comment]
> Please keep `show_research` or move it to batchflow

Doesn't `plot_results_by_config` do the same thing?

**Comment by a-arefina** [comment]
> > Please keep `show_research` or move it to batchflow
> 
> Doesn't `plot_results_by_config` do the same thing?

`show_research` plots graphs for all configs on a single plot.  `plot_results_by_config` has a separate plot for each config

**Comment by anton-br** [comment]
> > > Please keep `show_research` or move it to batchflow
> > 
> > 
> > Doesn't `plot_results_by_config` do the same thing?
> 
> `show_research` plots graphs for all configs on a single plot. `plot_results_by_config` has a separate plot for each config

okay, i will leave it for how, but i think it is better to move this action to batchflow

**Comment by anton-br** [comment]
> Please keep `show_research` or move it to batchflow

my mistake. `show_research` is already in batchflow https://github.com/analysiscenter/batchflow/blob/1ca06869b935dcffd8570328824702daf50ada9a/batchflow/utils.py#L130

File: models/First_break_picking/1d_CNN/research.ipynb [hunk 1]
```diff
@@ -47,9 +47,9 @@
     "\n",
     "from seismicpro.batchflow import Pipeline, B, V, C, W\n",
     "from seismicpro.batchflow.models.torch import UNet\n",
-    "from seismicpro.src import (SeismicDataset, FieldIndex, TraceIndex, seismic_plot,\n",
-    "                            show_research, print_results)\n",
-    "from seismicpro.batchflow.research import Research, Option"
+    "from seismicpro.batchflow.utils import print_results, show_research\n",
```
Comment (a-arefina) [inline models/First_break_picking/1d_CNN/research.ipynb:50]: Did you recalculate the notebooks to check that everything works after changing import sources?

File: seismicpro/src/__init__.py [hunk 2]
```diff
@@ -4,6 +4,6 @@
                             SegyFilesIndex, CustomIndex, KNNIndex)
 from .seismic_dataset import SeismicDataset
 
-from .plot_utils import spectrum_plot, seismic_plot, statistics_plot, gain_plot
+from .plot_utils import spectrum_plot, seismic_plot, statistics_plot, gain_plot, draw_histogram
```
Comment (a-arefina) [inline seismicpro/src/__init__.py:7]: Please provide a notebook with a usage example of this feature. When do we need to plot such histograms?

Comment (anton-br) [reply seismicpro/src/__init__.py:7]: this function is used in https://github.com/gazprom-neft/SeismicPro/blob/master/models/Ground-roll_attenuation/Unet_1D_model/model_estimation.ipynb i just returned it back

**Review by dpodvyaznikov — COMMENTED** [review]
Update notebooks with research and it'll be ready.

**Comment by anton-br** [comment]
all here is done, please approve or leave your comments
@a-arefina @dpodvyaznikov @nikita-klsh

**Review by nikita-klsh — APPROVED** [review]

**Pull request merged**